src/nodejs/util/network.js

```diff
@@ -0,0 +1,117 @@
+// network.js
+
+// network helper functions
+
+require('module-alias/register')
+
+const _       = require('underscore')
+
+const dbg     = require('nodejs_util/debug')
+var neon      = require('@cityofzion/neon-js')
+
+var cfg       = require('nodejs_config/config.js')
+var config    = cfg.load('nodejs_config/nodejs.config.json')
+
+let node  = ''
+let defly = false
+
+function print(msg) {
+  console.log(msg);
+}
```

Comment:
another print declaration function, could this be extracted?

Reply:
I like the macro. I'm planning to move it to the debug module in a future PR though.